Source/Core/Core/IOS/DolphinDevice.cpp

```diff
@@ -0,0 +1,81 @@
+// Copyright 2019 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include <sys/time.h>
```

Comment:
This doesn't appear to be necessary (as far as I can tell). `<algorithm>` and `<cstring>` need to be included though to satisfy `std::min` and `std::strlen`